clients/venice-producer/src/main/java/com/linkedin/venice/producer/online/ProducerTool.java

```diff
@@ -198,7 +198,8 @@ private static Object getValueObject(String valueString, RouterBasedStoreSchemaF
         value = adaptDataToSchema(valueString, valueSchema);
         break;
       } catch (Exception e) {
-        // Nothing to do. Try the next schema
```

Comment:
This is too verbose as it will print exceptions even if a schema could be matched. I have another PR (#626) for this. Can we use that instead?